src/Microsoft.Health.Fhir.Core/Features/Search/SearchValues/StringSearchValue.cs

```diff
@@ -42,7 +42,7 @@ public static StringSearchValue Parse(string s)
         {
             EnsureArg.IsNotNullOrWhiteSpace(s, nameof(s));
 
-            return new StringSearchValue(s);
+            return new StringSearchValue(s.UnescapeSearchParameterValue());
```

Comment:
Does/should this have an effect on https://github.com/microsoft/fhir-server/blob/master/src/Microsoft.Health.Fhir.Core.UnitTests/Features/Search/SearchValues/StringSearchValueTests.cs#L65?

Can we add a test for `StringSearchValue.String` to verify this behavior?

Reply:
Added.

Reply:
Thanks, and good point. Is there anything we can do that would make this more obvious? Would updating the comment on `String` and `ToString` help?

Reply:
I've update comment on top of `String` property. Not sure that changes saved me in past of been confused, but that confusion is purely based on reusing word `String` to many times in class :)